cinema/views.py

```diff
@@ -0,0 +1,35 @@
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+from rest_framework.generics import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
```

Comment:
Order of imports is next:
- build-in
- blank line
- third-party
- blank line
- custom
- two blank lines
- your code